Makefile

```diff
@@ -74,8 +76,10 @@ ifeq ($(shell which goveralls),)
 	@go get github.com/mattn/goveralls
 endif
 ifeq ($(shell which counterfeiter),)
-	@echo "Installing counterfeither..."
+	@echo "Installing counterfeither $(COUNTERFEITER)..."
 	@go get github.com/maxbrunsfeld/counterfeiter
+	@cd ${GOPATH}/src/github.com/maxbrunsfeld/counterfeiter && git checkout tags/$(COUNTERFEITER) && go install
```

Comment:
I think it would be better to put lines 81 and 82 in the generate target so that we make sure this version is used on each generation